src/config.ts

```diff
@@ -327,6 +327,23 @@ export const META_TX_WORKER_REGISTRY: string | undefined = _.isEmpty(process.env
     ? undefined
     : assertEnvVarType('META_TX_WORKER_REGISTRY', process.env.META_TX_WORKER_REGISTRY, EnvVarType.ETHAddressHex);
 
+export const RFQM_META_TX_SQS_URL: string | undefined = _.isEmpty(process.env.RFQM_META_TX_SQS_URL)
+    ? undefined
+    : assertEnvVarType('RFQM_META_TX_SQS_URL', process.env.RFQM_META_TX_SQS_URL, EnvVarType.Url);
+
+// AWS Configs
+export const AWS_REGION: string | undefined = _.isEmpty(process.env.AWS_REGION)
```

Comment:
We don't need these, because they will come automagically from role-based authentication, right @opaolini ?

Reply:
would be great to remove these! Let's see what Oskar says :D 

Although, one consideration I had was on how do we run it locally without env variables - right now I put these here to make it easier to run locally, but there might be a better way

Reply:
The SDKs do some automatic region determination when running on AWS EC2 and/or Lambda [0], also the SDK will check the `AWS_REGION` env variable on its own, so no need to specify it here. That's also true for all the AWS client configuration parameters actually.

IAM Role will solve the authentication so `ACCESS_KEY` and `SECRET_ACCESS_KEY`won't need to be passed in.

[0] - https://docs.amazonaws.cn/sdk-for-java/latest/developer-guide/region-selection.html